openapiart/openapiartgo.py

```diff
@@ -600,20 +628,40 @@ def _write_interface(self, new):
             }}
 
             func (obj *{struct}) ToPbText() string {{
+                obj.Validate()
+                vErr := validationResult()
+                if vErr != nil {{
+                    panic(vErr)
```

Comment:
@ajbalogh @ashutshkumr , is it okay to panic for all To<Pb, Json, YAML> methods with in the framework or do we have to return string and error?

Reply:
I like the panic as it allows for the return to be used directly in methods like fmt.Println().
If we change the To<Pb, Json, Yaml> methods to return a tuple then we should add the Stringer interface to allow for a non-validation dump method.